src/Sylius/Bundle/ApiBundle/Changer/PaymentMethodChanger.php

```diff
@@ -31,7 +31,7 @@ public function __construct(
 
     public function changePaymentMethod(
         string $paymentMethodCode,
-        string $paymentId,
+        int $paymentId,
```

Comment:
```suggestion
        int|string $paymentId,
```
I'm nor really sure about that lets make it like that or even `mixed`

Reply:
But I changed the variable in the command from string to int, so it will always be int.

Reply:
The `id` should not be typehinted only to `int` also in command